docs/content/preview/releases/techadvisories/ta-27380.md

```diff
@@ -0,0 +1,98 @@
+
+A few additional considerations to assess impact are:
+
+- Range sharding is currently not enabled by default, so if a table/index is created without any qualifiers on the primary key, it is hash-sharded by default.
+
+- YCQL tables cannot be created with [range sharding](../../../architecture/docdb-sharding/sharding/#range-sharding).
+
+- To explicitly create a range-sharded table/index, you would have to specify ASC or DESC as part of the primary key specification as described in the following example:
+
+  ```sql
+  CREATE TABLE range(k1 int,
+                     k2 int,
+                     v1 int,
+                     v2 text,
+                     PRIMARY KEY (k1 ASC, k2 DESC));
+  ```
+
+- Range sharding is also enabled by default for new tables with the pg_parity flags.
+- Default [tablet](../../../architecture/key-concepts/#tablet) count with range sharding is 1, so if [automatic tablet splitting](../../../architecture/docdb-sharding/tablet-splitting/#automatic-tablet-splitting) is NOT enabled, then you will not experience this issue. Note that tablet splitting is enabled by default.
+
+To identify whether you are impacted:
+
+- The problem occurs almost always when a range partitioned table has multiple tablets. The only scenario where it may not happen is if all writes are going to a single tablet which is unlikely. Another unlikely case is where the workload is not using transactions at all.
+
+- For tables, row counts can be compared on both source and target tables. As the rows on the target are lost permanently, the inconsistency is permanent.
+
+- For hash sharded tables with range sharded indexes, the row count may match as it scans the table, but an index only scan would show a difference between the source and target tables.
+
+- While depending on the key type, it might be possible to identify exactly which key range/tablet is correctly replicated and the remaining ones would be missing data - this may not really help because all tablets except 1 tablet would be missing data.
+
+- The issue is caused by an implementation bug in determining whether a given key belongs to the tablet's key ranges during the apply of transactions on the xCluster target.
+
+## Mitigation
+
+The mitigation is to fully rebootstrap the xCluster target after applying the fix or using one of the following alternative approaches. If you're using range sharding with xCluster, it is recommended that you apply the fix to prevent the issue from happening.
+
+Alternatives to avoid encountering the issue:
+
+- Use hash sharding instead of range sharding, if that is a viable option.
+- Use a single tablet for the range sharded table/index on the xCluster target and disable Automatic Tablet Splitting to avoid creating multiple tablets.
```

Comment:
should we link automatic tablet splitting to a page we have?

Reply:
already linked it for its first instance on the page.